chain/client/src/sync.rs

```diff
@@ -41,6 +41,9 @@ pub const MAX_BLOCK_HEADERS: u64 = 512;
 /// Maximum number of block header hashes to send as part of a locator.
 pub const MAX_BLOCK_HEADER_HASHES: usize = 20;
 
+/// Maximum number of block requested at once in BlockSync
+const MAX_BLOCK_REQUESTS: usize = 3;
```

Comment:
Why 3?

Reply:
Requesting many blocks in advance won't help that much because the bottleneck is at requesting chunks. Since now we know that we can't afford to request chunks for 5 blocks, setting this number to >= 5 doesn't make much sense. Out of the numbers that are >=2 but < 5, I chose 3 since it's the middle